proposed-decisions.md

```diff
@@ -0,0 +1,128 @@
+
+Entitlement will be **sufficient** to ensure valid and transparent transactions
+
+## Blended learning as base design criteria
+[Related Issue 11](https://github.com/stichtingsem/functional-overview/issues/11)
+
+Blended learning (combi's) is **not** the base design criteria, but asks for additional requirements to partners in the ecosystem
+
+- Flexibility in delivery
+    - Digital components are (near) real time accessible
+    - Shortage of folio components can;t block the access to digital components
+- Blended learning materials (combi’s/LiFo) remain one product number (EAN)
+- Entitlements (digital) and purchase orders (folio) need to be synced periodic
+- Entitlement ensures access for users
+- Entitlement guarantees transactions for partners
+
+## Revenue recognition
+[Related issue 12](https://github.com/stichtingsem/functional-overview/issues/12)
+
+[Related issue 13](https://github.com/stichtingsem/functional-overview/issues/13)
+
+Revenue recognition is influenced by tax legislation, competitive propositions and commercial contracts/agreements. Partners in the ecosystem should determine their own approach to revenue recognition, but the services should ensure valid & transparent transactions and enable various transaction models
+
+## Identity
+[Related Issue 14](https://github.com/stichtingsem/functional-overview/issues/14)
+
+- Product: EAN
+- User: ECK-ID
+- School: Digi Delivery ID (RIO will be evaluated on applicability)
+
+## Bring SIS (LAS) and LML in sync
+[Related issue 15](https://github.com/stichtingsem/functional-overview/issues/15)
+
+SIS (LMS) data will be synced with LML
+
+Boundaries:
+- Target binding based on role
+- School is in control of the accessibility to the entities/data within SIS
+
+Preparation phase:
+- Subjects
+- Courses
+
+Execution phase:
+- Forecast (base for entitlement)
+
+## First Time Right
+[Related Issue 16](https://github.com/stichtingsem/functional-overview/issues/16)
+
+First Time Right will **not** be a principle, but there are (additional) requirements:
+- Avoidance of errors and repairs
+    - Determine when data is up to date and ready for use (data quality)
+- Updates (LML) will be shared (near) real time
+    - It is a dynamic process and updates will prevent errors (and repairs)
+    
+## In Scope POC
+[Related Issue 17](https://github.com/stichtingsem/functional-overview/issues/17)
+
+### In Scope:
+- VO (secondary) and MBO (vocational)
+- Full Digital (100% digital)
+- Blended (combi’s, LiFo)
+- Folio (incl. paper vouchers)
+    - Partners (SA, MP) within the eco-system have to evaluate the impact of the proposed solutions on their supply chain (forecasts/prognosis, purchase, stock, etc)
+- ECK
+
+[Related Issue 18](https://github.com/stichtingsem/functional-overview/issues/18)
+
+### Out of scope (for now):
+- Setup & Support
+    - Most important next step in the SEM pilot
+    - More decentralized = more risk > How to reduce risk?
```

Comment:
I don't agree with the notion of decentralization equals more risk, quite the reverse (it's also out of scope, but not sure what risk you're referring to).

Reply:
```suggestion
    - More decentralization asks for higher maturity level regarding to maintenance and fixation of stem data and identity 
      - How to guard data quality?
```